utils/polus-imagej-util/generate.py

```diff
@@ -0,0 +1,138 @@
+import os
+import jpype
+import imagej
+import shutil
+import classes.populate as cp
+from pathlib import Path
+
+
+"""This file uses the classes in populate.py and cookiecutter to automatically parse the ImageJ ops help and create plugins"""
+
+if __name__ == '__main__':
+
+    print('Starting JVM\n')
+    
+    # Start JVM
+    ij = imagej.init('sc.fiji:fiji:2.1.1+net.imagej:imagej-legacy:0.37.4', headless=True)
+    
+    # # Retreive all available operations from pyimagej
+    # imagej_help_docs = scyjava.to_python(ij.op().help())
+    # print(imagej_help_docs)
+    
+    print('Parsing imagej ops help\n')
+    
+    # Populate ops by parsing the imagej operations help
+    populater = cp.Populate(ij)
```

Comment:
Can we just have the `Populate` function deal with `jpype`? We can just create a function that starts `jpype`, initializes ImageJ, generates all the cookiecutter jsons, etc.

I say this because we are doing some manual work here that could be self-contained in `populate.py`, and then the entire purpose of this file could just be a simple loop inside a bash script.

The only thing we would need to do outside of this is automate the unit testing, but that could be part of a `test.sh` bash script.

Reply:
I have changed this so that `Populate` instantiates an `imagej` instance when it is instantiated itself.